src/common/runtime/server.ts

```diff
@@ -38,10 +39,22 @@ function usage(rc: number): never {
 interface RunResult {
   status: string;
   message: string;
+  coverageData?: string;
 }
 
+// The interface that exposes creation of the GPU, and optional interface to code coverage.
 interface GPUProviderModule {
+  // @returns a GPU with the given flags
   create(flags: string[]): GPU;
+  // An optional interface to a CodeCoverageProvider
+  coverage?: CodeCoverageProvider;
+}
+
+interface CodeCoverageProvider {
+  // Starts collecting code coverage
+  begin(): void;
+  // Ends collecting of code coverage, returning the coverage data
```

Comment:
If I read dawn.node's code correctly, it doesn't return the coverage data but the path to the coverage data.

Reply:
Added a comment to make clear that it's opaque